blog/2024-05-03-release-notes.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Luigi v2.12
+seoMetaDescription: Release notes for Luigi v2.12
+author:
+  - Mahati Shankar
+layout: blog
+---
+
+You can read about the new features in Luigi v2.12 in the release notes below.
+
+<!-- Excerpt -->
+
+
+#### Opt Out of Luigi Error Handling for 404s
+
+If you have implemented your own `page not found` error handler function, then you can set [**ignoreLuigiErrorHandling**](https://docs.luigi-project.io/docs/navigation-parameters-reference?section=pagenotfoundhandler) to `true` in your routing configuration.
+
+#### Update Context via Core API
+
+This addition lets you update the context values for visible iframes and Luigi web components.
+
+#### Change Luigi Core Version on the Fly in Luigi Fiddle
```

Comment:
this is a feature in luigi-fiddle, not luigi itself, so should not be part of the luigi release notes